utils_cv/detection/dataset.py

```diff
@@ -34,7 +35,7 @@ def get_transform(train: bool) -> List[object]:
     """
     transforms = []
     transforms.append(ToTensor())
-    if train:
+    if train is True:
         transforms.append(RandomHorizontalFlip(0.5))
```

Comment:
Could you add this transformation here always for train:
torchvision.transforms.ColorJitter(brightness=0.2, contrast=0.2, saturation=0.4, hue=0.05)

Reply:
done

Reply:
done

Reply:
thanks!